src/lib.rs

```diff
@@ -54,6 +69,15 @@ pub fn perform_cracking(text: &str, config: Config) -> Option<DecoderResult> {
     searchers::search_for_plaintext(text)
 }
 
+fn check_if_input_text_is_plaintext(text: &str) -> bool {
+    let athena_checker = Checker::<Athena>::new();
+    athena_checker.check(text).is_identified
+}
+
+fn return_early_because_input_text_is_plaintext() {
+    println!("Your input text is the plaintext 🥳")
```

Comment:
Shouldn't this be in verbose mode? Why are we doing `println!`?

Reply:
Good point, this would be ugly in the API -- I'll clean it all up in a centralised printing module, I'll make macros for being able to tell if we're in the CLI or API :)